src/Settings.php

```diff
@@ -406,6 +406,16 @@ public function __construct()
                             'type' => 'text',
                         ],
                     ],
+                    [
+                        'name' => __('Apply social sharing options', 'planet4-master-theme-backend'),
+                        'id' => 'apply_social_sharing_options',
+                        'type' => 'radio',
+                        'options' => [
+                            'posts' => __('Posts', 'planet4-master-theme-backend'),
+                            'posts_and_forms' => __('Posts & Forms', 'planet4-master-theme-backend'),
+                        ],
+                        'default' => 'posts_and_forms',
+                    ],
```

Comment:
Maybe move the new option under the `social_share_options`, so it's more clear what it refers to

Reply:
Done!